howso/engine/trainee.py

```diff
@@ -1343,6 +1343,13 @@ def react(
                 features of the reacted case to determine that area. Relies on
                 'robust_influences' parameter to determine whether to do
                 standard or robust computation.
+            - features: list of str, optional
```

Comment:
features :

Reply:
Fixing

Reply:
After discussion with @jdbeel, I decided to just add all the spaces to the client react details docstring as well.